doc/tutorial/statistical_inference/unsupervised_learning.rst

```diff
@@ -41,34 +36,24 @@ algorithms. The simplest clustering algorithm is :ref:`k_means`.
     >>> print(y_iris[::10])
     [0 0 0 0 0 1 1 1 1 1 2 2 2 2 2]
 
+.. figure:: /auto_examples/cluster/images/sphx_glr_plot_cluster_iris_001.png
+   :target: ../../auto_examples/cluster/plot_cluster_iris.html
+   :scale: 63
+
 .. warning::
 
     There is absolutely no guarantee of recovering a ground truth. First,
     choosing the right number of clusters is hard. Second, the algorithm
     is sensitive to initialization, and can fall into local minima,
     although scikit-learn employs several tricks to mitigate this issue.
 
-    |
-
-    .. figure:: /auto_examples/cluster/images/sphx_glr_plot_cluster_iris_003.png
-       :target: ../../auto_examples/cluster/plot_cluster_iris.html
-       :scale: 63
-
-       **Bad initialization**
-
-    .. figure:: /auto_examples/cluster/images/sphx_glr_plot_cluster_iris_001.png
-       :target: ../../auto_examples/cluster/plot_cluster_iris.html
-       :scale: 63
-
-       **8 clusters**
-
-    .. figure:: /auto_examples/cluster/images/sphx_glr_plot_cluster_iris_004.png
-       :target: ../../auto_examples/cluster/plot_cluster_iris.html
-       :scale: 63
-
-       **Ground truth**
+    For instance, on the image above, we can observe the difference between the
+    ground-truth (bottom right figure) and different clustering. We do not
+    recover the expected labels, either because the number of cluster was
+    chosen to be to large (top left figure) or suffer from a bad initialization
+    (top right figure).
```

Comment:
"bottom left" and "top right" need swapping to match the labeling in the figure.